src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,22 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
```

Comment:
Please recheck if any IDE plugin or something reformats existing code. These changes look weird.
Pull request shouldn't contain changes it's not designed for